packages/flutter_tools/lib/src/build_runner/web_fs.dart

```diff
@@ -111,6 +113,7 @@ class WebFs {
   final String _target;
   final BuildInfo _buildInfo;
   final bool _initializePlatform;
+  final List<String> dartDefines;
```

Comment:
Should we make `dartDefines` private here to match with everything else?

Reply:
Good catch! Done.